robot/Cumulus/tests/browser/batch_gift_entry/opp_autoclosed.robot

```diff
@@ -0,0 +1,65 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library           DateTime
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Opportunity is Autoclosed when Overpaid
+    [Documentation]    Create Open Opportunity and apply new payment of amount more than opportunity amount and confirm that opportunity is closed when batch is processed
+    [tags]  unstable
```

Comment:
Just checking in: could this test be marked stable?

Reply:
yes thanks for reminding. Also, I updated tag for the test in stable flow to unstable that has been failing on and off